backend/de.metas.business/src/main/java/de/metas/invoice/service/impl/AbstractInvoiceDAO.java

```diff
@@ -502,6 +502,21 @@ public <T extends org.compiere.model.I_C_Invoice> List<T> getByDocumentNo(final
 				.create()
 				.list(modelClass);
 	}
+	
+	@Override
+	public List<I_C_Invoice> retrieveUnpaid(
```

Comment:
why not using Sets instead of List?
also, what shall the method return in case docNos or docStatuses is empty?

Reply:
changed to Sets
in the current implementation (after review) the method returns all unpaid invoices (when both docNos and docStatuses are empty).

Reply:
saw that u changed the returned type `List<I_C_Invoice>` to Set... leave it like that,
but basically i was referring to params